docs/core/entity/update.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Update Entity
+sidebar_label: Update
+---
+
+An update entity is an entity that indicates if an update is available for a
+device or service. This can be any update, including update of a firmware
+for a device like a light bulb or router, or software updates for things like
+add-ons or containers.
+
+It can be used for:
+
+- Providing an indicator if an update is available for a device or service.
+- An install method to allow installing an update or a specific version
+  of the software.
+- Allow for offering backups before installing a new update.
+
+## Properties
+
+:::tip
+Properties should always only return information from memory and not do I/O (like network requests). Implement `update()` or `async_update()` to fetch data.
+:::
+
+| Name | Type | Default | Description
+| ---- | ---- | ------- | -----------
+| current_version | str | `None` | The current version of the software installed.
+| in_progress | bool, int | `None` | Update installation progress. Can either return a boolean (True if in progress, False if not) or an integer to indicate the progress in from 0 to 100%.
+| lastest_version | str | `None` | The latest version of the software available.
+| release_summary | str | `None` | Summary of the release notes or changelog. This is not suitable for long changelogs but merely suitable for a short excerpt update description of max 255 characters.
+| release_url | str | `None` | URL to the full release notes of the latest version available.
+| title | str | `None` | Title of the software. This helps to differentiate between the device or entity name versus the title of the software installed.
+
+Other properties that are common to all entities such as `device_class`, `entity_category`, `icon`, `name` etc are still applicable.
+
+## Supported Features
+
+| Constant | Description |
+|----------|--------------------------------------|
+| 'SUPPORT_BACKUP' | A backup can be made automatically, before installing an update.
+| 'SUPPORT_INSTALL' | The update can be installed from Home Assistant.
+| 'SUPPORT_PROGRESS' | This integration is able to provide progress information
+| 'SUPPORT_SPECIFIC_VERSION' | A specific version of an update can be installed using the `update.install` service.
+althought is it better if it can be extracted from the device or service API.
+on the installation process. If omitted, Home Assistant will try to provide,
```

Comment:
Try to provide? looks like the last part of the sentence is missing here.

Reply:
Hehe, I sorted the table, causing the sentences to mix up. Fixed 👍